core/src/main/java/com/linecorp/armeria/client/logging/LoggingClientBuilder.java

```diff
@@ -223,4 +223,9 @@ public LoggingClientBuilder responseCauseSanitizer(
                     ? extends @Nullable Object> responseCauseSanitizer) {
         return (LoggingClientBuilder) super.responseCauseSanitizer(responseCauseSanitizer);
     }
+
+    @Override
+    public LoggingClientBuilder addExpectedException(Class<? extends Throwable> clazz, LogLevel logLevel) {
```

Comment:
Meanwhile, `ResponseLogLevelMapper` is introduced. Could we move this feature to `ResponseLogLevelMapper`?
```
interface RequestLogLevelMapper {
      static of(Class<? extends Throwable> clazz, LogLevel logLevel) {

      }

      // Optionally, we can add a builder for fluently creating a LogLevel mapper.
}
```

Reply:
It seems good.

Reply:
On second thought, this API is vague whether it is applied to `requestCause()` and `responseCause()`.
How about removing this API and `RequestLogLevelMapper.of(Class<? extends Throwable> clazz, LogLevel logLevel)`.
Everything else is good to me.

Reply:
@ikhoon 
Let me check on your proposal.
Is it correct that the following two points should be addressed?

- `addExpectedException` would be defined only in `LoggingServiceBuilder`
- `RequestLogLevelMapper.of(Class<? extends Throwable> clazz, LogLevel logLevel)` doesn't exist now, Does this refer to `ResponseLogLevelMapper.of(Class<? extends Throwable> clazz, LogLevel logLevel)`?

Reply:
I'd like to refactor the current implementation like:
- Remove `addExpectedException`. It would be better to add the new API only to `{Request,Response}LogLevelMapper`.
- Add `RequestLogLevelMapper.of(Class<? extends Throwable> clazz, LogLevel logLevel)` and `ResponseLogLevelMapper.of(Class<? extends Throwable> clazz, LogLevel logLevel)`.
- Optionally, add `ResponseLogLevelMapper.builder()` for providing fluent APIs. We may handle this feature in the following PR or if there is a demand for it.

Reply:
Thanks for your detailed explanation!

Reply:
@ikhoon 
I addressed them.
Please review again.